tools/functional-tester/etcd-tester/main.go

```diff
@@ -53,6 +53,7 @@ func main() {
 	stresserType := flag.String("stresser", "keys,lease", "comma separated list of stressers (keys, lease, v2keys, nop, election-runner, watch-runner, lock-racer-runner, lease-runner).")
 	etcdRunnerPath := flag.String("etcd-runner", "", "specify a path of etcd runner binary")
 	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
+	failpoints := flag.String("failpoints", `panic("etcd-tester")`, `define failpoints (e.g. concat of panic("etcd-tester") and 1*sleep(1000)->panic("etcd-tester")`)
```

Comment:
`comma separated list of failpoint terms to inject (e.g. 'panic("etcd-tester"),1*sleep(1000)')`